packages/contracts-node/package.json

```diff
@@ -33,8 +32,7 @@
   "scripts": {
     "test": "echo 'No tests configured'",
     "clean": "rm -rf dist generated typechain",
-    "copy-across-types": "mkdir -p typechain/@across-protocol/contracts && rsync -a $(dirname $(node -p 'require.resolve(\"@across-protocol/contracts/package.json\")'))/contract-types/ typechain/@across-protocol/contracts",
-    "generate-ts": "yarn clean && mkdir -p generated typechain/core && cp -R ../core/contract-types/* typechain/core/ && yarn copy-across-types && yarn hardhat generate-contracts-node --out ./generated/index.ts",
+    "generate-ts": "yarn clean && mkdir -p generated typechain/core && cp -R ../core/contract-types/* typechain/core/ && yarn hardhat generate-contracts-node --out ./generated/index.ts",
```

Comment:
@mrice32 Something in the `generate-ts` target seems to have an underlying dependency on `copy-across-types` having been successfully run, because otherwise we fail on trying to sync some directories ([fails CI with ENOENT](https://app.circleci.com/pipelines/github/UMAprotocol/protocol/21541/workflows/5df7b8cf-d071-4b66-a5b7-a02e174c87f0/jobs/194178)).

I'm not really sure how to resolve it..my inclination is just to keep removing things, but it seems like some `copy-<package>-types` target might actually be required (also for use as an example). Should we perhaps find a substitute package that can be used in place of `@across-protocol/contracts`?